src/views/InvolvementProfile/index.jsx

```diff
@@ -303,10 +297,9 @@ const InvolvementProfile = () => {
                       <div className={styles.photoUploader} {...getRootProps()}>
                         <input {...getInputProps()} />
                         <img
-                          className="rounded_corners"
+                          className={'rounded_corners' + styles._imageDrop}
```

Comment:
What is this doing?  is this actually referencing a CSS class called rounded_corners_imageDrop(which is what 'rounded_corners' + styles._imageDrop is doing)?